core/src/UserSettingsEditor.html

```diff
@@ -59,6 +59,18 @@
         userSettings[userSettingsGroupSchema[0]][key] = option;
     }
 
+    function updateEnumButton(key, option) {
+      document.querySelectorAll('.enum-buttons-container-'+ key  +' button').forEach(button => {
+        let buttonTestid = button.getAttribute('data-testid');
```

Comment:
I am not a fan of using test id here. Test Id is for cypress only. Maybe use custom data attributes instead?

Reply:
oh my bad, I was not to see it clearly. It already used data attribute here.

Reply:
Or give it an `id` and use it.

Reply:
@legteodav , I would suggest to have `id` attribute instead. Keep test id for cypress only. How do you think?

Reply:
Just changed it..... (sorry I forgot...)